Godeps/Godeps.json

```diff
@@ -1798,6 +1814,542 @@
 		{
 			"ImportPath": "speter.net/go/exp/math/dec/inf",
 			"Rev": "42ca6cd68aa922bc3f32f1e056e61b65945d9ad7"
+		},
+		{
+			"ImportPath": "github.com/Sirupsen/logrus/formatters/logstash",
+			"Comment": "v0.7.3",
+			"Rev": "aaf92c95712104318fc35409745f1533aa5ff327"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/awserr",
```

Comment:
This doesn't look right, usually all godeps are sorted by name, you have here disordered deps. Are you sure you properly used godes?